test/testautomation_stdlib.c

```diff
@@ -1188,6 +1188,39 @@ stdlib_iconv(void *arg)
     return TEST_COMPLETED;
 }
 
+
+static int
+stdlib_strpbrk(void *arg)
+{
+    struct {
+        const char *input;
+        const char *accept;
+        int expected; /* negative if NULL */
+    } test_cases[] = {
+        { "", "", -1 },
+        { "abc", "", -1 },
+        { "Abc", "a", -1 },
+        { "abc", "a", 0 },
+        { "a;b;c", ";", 1 },
+        { "a;b;c", ",", -1 },
+        { "a:b;c", ";:", 1 },
+        { "Hello\tSDL", " \r\n\t", 5 },
```

Comment:
Just a personal suggestion: it might be worth adding a test case where the character that is found isn't at the last position in the breakset/accept list, as that would account for loops over characters in accept() that forget to `break;` (or similar) when a match is found, and end up mistakenly overwriting the match boolean with latter characters and fail to detect a match.

Reply:
I'm going to remove it from the series, and propose it separately.
I'll tag you for proposing extra examples ;)

Reply:
https://github.com/libsdl-org/SDL/pull/10695 Feel free to comment :)